src/Cat.java

```diff
@@ -0,0 +1,21 @@
+public class Cat extends Animal {
```

Comment:
If you haven't implemented this method, it means that an instance of class `Cat` can still call method `swim()` from `Animal` class, and a message will be printed to the console, which is incorrect. 
There is a requirement in a task description, that cats cannot swim, which means you should override the method `swim()` from the superclass `Animal` and add some message that will be printed to console, that cats cannot swim.